docs/source/conf.py

```diff
@@ -87,7 +94,7 @@
 
 # The name for this set of Sphinx documents.  If None, it defaults to
 # "<project> v<release> documentation".
-html_title = "TraitsUI 4 User Manual"
+html_title = "TraitsUI 5 User Manual"
```

Comment:
yes. added a new commit with the aforementioned corrections/changes. will look into using |version| now